### ISSUE TITLE
pq\DateTime::__toString return type

### ISSUE THREAD

**Issue by siilike** [m6w6/ext-pq#46]
Under PHP 8.2 this extension logs a notice:
```
NOTICE: PHP message: PHP Warning:  pq\DateTime::__toString() implemented without string return type in Unknown on line 0
```

**Comment by siilike** [comment]
I can see that this has been fixed in commit 64cc270, but no release with it has been made.

**Comment by m6w6** [comment]
Thanks for the heads up & sorry for the delay!
